meta/client/tidbclient/user.go

```diff
@@ -26,6 +26,26 @@ func (t *TidbClient) GetUserBuckets(userId string) (buckets []string, err error)
 	return
 }
 
+func (t *TidbClient) GetAllUserBuckets() (bucketUser map[string]string, err error) {
```

Comment:
I think it is not necessary.  The `userId` (or called `projectId`)  will be confirmed by IAM request (or from memory) when 
 handling PutObject.

Reply:
Because
- we need to handle anonymous users
- it's faster to query memory than redis
- it leads to better code structure